build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -xe
+set +o pipefail
+
+driver_images_tag_from_branch=latest
+triggering_branch=$CI_ACTION_REF_NAME
+target_image_tags=`build/ci/get_image_tags_from_branch.sh ${triggering_branch}`
+target_specific_tag=`echo $target_image_tags | awk '{print$2}'`
+
+is_private_branch_component_image_exists(){
+  driver_component=$1
+  is_image_tag_exists=false
+  export driver_image_inspect=`docker manifest inspect $csiblock_docker_registry_username/ibm-block-csi-$driver_component:$target_specific_tag`
+  if [[ "$driver_image_inspect" != "" ]]; then
+    is_image_tag_exists=true
+  fi
+  echo $is_image_tag_exists
```

Comment:
consider using a [shorter command](https://stackoverflow.com/a/52077346) with /dev/null and just checking the exit status

Reply:
done

Reply:
do you mean move [those](https://github.com/IBM/ibm-block-csi-operator/blob/ab2840316face7631cfccf0e3e45cb545a745f44/build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh#L19-L24) lines into [is_private_branch_component_image_exists](https://github.com/IBM/ibm-block-csi-operator/blob/ab2840316face7631cfccf0e3e45cb545a745f44/build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh#L9) function?

Reply:
no, I mean restore the function to how it was before (echoing true/false), but according to `$? == "0"`, so the caller would continue to compare using ` == "true"`

Reply:
hope it's clear